features/FEATURE_BLE/source/generic/GenericSecurityManager.cpp

```diff
@@ -750,8 +768,51 @@ ble_error_t GenericSecurityManager::oobReceived(
 // Helper functions
 //
 
+ble_error_t GenericSecurityManager::init_database(
```

Comment:
this is just moved

Reply:
That's perfect to move it into a function; it makes code more readable.